src/v3/src/transformer/transaction/transform.ts

```diff
@@ -10,11 +10,17 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { IdxRemediation } from '@okta/okta-auth-js';
-import { flow } from 'lodash';
```

Comment:
I _think_ this actually is the preferred way to import from `lodash` now: https://lodash.com/per-method-packages